.circleci/config.yml

```diff
@@ -1,9 +1,16 @@
+version: 2.1
+
+executors:
+  bionic:
+    docker:
+      - image: buildpack-deps:bionic
+    environment:
+      LANG: "C.UTF-8"
+      EMTEST_DETECT_TEMPFILE_LEAKS: "1"
+      EMCC_CORES: "8"
```

Comment:
It looks like the actual parallelism is 2 despite what `/proc` says: https://circleci.com/docs/2.0/configuration-reference/#resource_class

Reply:
I would suggest increase the parallelism of build-upstream if possible, since it's a bottleneck that blocks all the upstream tests.

Reply:
we could set `resource_class: large` to run that step on a bigger VM. Assuming we can saturate all 4 cores and come close to going twice as fast, it wouldn't even cost extra.

Reply:
I'm a little confused because /proc/cpuinfo on the circle machines seems to think the machine has 36 cores already :)  But we run of out resources doign a fork() if we try to use all of those.

Why would it not cost more?

Reply:
Because if the parallelization is perfect, then we run for half the time and it cancels out.

Reply:
So the price per minute is more expensive for us on the `resource_class: large` but we could hope to use fewer minutes?   We can experiment with that as a followup.

Reply:
According to the note in the [doc](https://circleci.com/docs/2.0/configuration-reference/#resource_class) `/proc/cpuinfo` is a lie.

And yes, the larger VM has 2x as many cores and costs exactly twice as much per minute. Of course the non-compiling parts of the run would not run 2x as fast (maybe 20% of the `build-upstream` build time) but it might be worth it.